src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -172,8 +172,8 @@
 {%- endblock dateinterval_widget -%}
 
 {%- block number_widget -%}
-    {# type="number" doesn't work with floats in localized formats #}
-    {%- set type = type|default('text') -%}
+    {%- set type = type|default('number') -%}
+    {%- set attr = attr|merge({step: 'any'}) -%}
```

Comment:
The `step` attribute should only be used when the `type` is set to `number`, right?

Reply:
Yes good point, I'm gonna move it to the html5 option as talked beneath.